experiments/evaluation/README.md

```diff
@@ -0,0 +1,18 @@
+This folder contains scripts for evaluating `theseus`'s components.
```

Comment:
Nice! Maybe also add a line about what each .sh script does and if there are any params involved in running the scripts.

Reply:
Oh I see each script has starting comment with this info. Might still be useful to have in the readme though.

Reply:
Added. @fantaosha can you confirm the descriptions for PGO look OK?